### ISSUE TITLE
 Is there no update to the Minecraft preview version 1.21.20.21? 

### ISSUE THREAD

**Issue by ForCris** [MCMrARM/mc-w10-version-launcher#303]
Is there no update to the Minecraft preview version 1.21.20.21?

**Comment by DavyCraft648** [comment]
try https://github.com/DavyCraft648/mc-w10-versiondb

**Comment by RavinMaddHatter** [comment]
Question... How do you get that setting
![image](https://github.com/MCMrARM/mc-w10-version-launcher/assets/4407668/299939c8-89ee-4e3c-a305-523f338978b5)

**Comment by RavinMaddHatter** [comment]
Question... how do you get that option 
![image](https://github.com/MCMrARM/mc-w10-version-launcher/assets/4407668/6366482c-0b0b-4284-b5e3-7dd0db483df5)

**Comment by DavyCraft648** [comment]
https://youtu.be/J6Jm6pT_C7k?si=SVCmAdaYWwPfBK4X

You need the latest version of MCLauncher

https://github.com/DavyCraft648/mc-w10-version-launcher/releases

**Comment by RavinMaddHatter** [comment]
Yeah it isnt in the complied release then... I grabed the latest release... i guess it is currently build your own...

**Comment by RavinMaddHatter** [comment]
Aah i see you forked and have a release thanks
